### DISCUSSION TITLE
0.23.7 released. Update required!

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/VWsFriend#436] [General]
You need to update in order to make VWsFriend working again with the backend

### fixed
- Changed URLs to the new URLs necessary to contact the backend

### Changed
- Updated API to 0.54.1

<hr /><em>This discussion was created from the release <a href='https://github.com/tillsteinbach/VWsFriend/releases/tag/v0.23.7'>0.23.7</a>.</em>

**Comment by Julien4135** [comment]
Hello, I do what command to know the new URLs after the update to go in the backend ?

**Reply by tillsteinbach** [reply]
Ah, this was badly phrased. The VWsFriend urls are still the same. The VW-Backend changed.

**Reply by Julien4135** [reply]
I'm sorry, but I still have a little bug :/ In fact I still have the remaining charge time gauge stuck at 1 hour? I've done the update and restarted.

**Reply by Julien4135** [reply]
The car synchronises well *

**Reply by Julien4135** [reply]
In the end, we only had to take the car to get it online.
Apparently everything is back in order.
Thanks 👍
